api/HTMLAllCollection.json

```diff
@@ -127,7 +127,7 @@
               "version_added": "10"
             },
             "safari_ios": {
-              "version_added": true
+              "version_added": "10"
```

Comment:
This is the `length` property. Pretty sure it was supported from the time the interface was introduced.